backend/docs/Login_with_Phone_guide.md

```
@@ -0,0 +1,230 @@
+source .env.local
+curl -X POST "https://api.twilio.com/2010-04-01/Accounts/$TWILIO_ACCOUNT_SID/Messages.json" \
+-u "$TWILIO_ACCOUNT_SID:$TWILIO_AUTH_TOKEN" \
+--data-urlencode "MessagingServiceSid=$TWILIO_MESSAGE_SERVICE_SID" \
+--data-urlencode "To=+917015538461" \
+--data-urlencode "Body=Your Disciplefy verification code is 123456. Do not share this code with anyone."
+```
+
+### **Expected Result**
+
+Users will receive SMS from Canada number:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)